src/Styleguide/Elements/Box.tsx

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from "styled-components"
+import { Flex, FlexProps } from "./Flex"
+import {
+  themeGet,
+  display,
+  DisplayProps,
+  space,
+  SpaceProps,
+} from "styled-system"
+
+const hover = css`
+  &:hover {
+    border-color: ${themeGet("colors.black60")};
+  }
+`
+
+export interface BorderBoxProps extends FlexProps {
+  hover?: boolean
+}
+
+export const BorderBox = styled(Flex).attrs<BorderBoxProps>({})`
```

Comment:
👍 - I was trying to figure this out earlier but didn't think to attach `.attrs` after! duhh